.github/workflows/python-package-conda.yml

```diff
@@ -0,0 +1,46 @@
+name: Conda-based pytest
+
+on: [push, pull_request]
+
+jobs:
+  build-linux:
+    runs-on: ubuntu-latest
+
+    strategy:
+      max-parallel: 5
+      matrix:
+        python-version: [3.6, 3.9]
+        slycot: ["", "conda"]
+        array-and-matrix: [0]
+        include:
+          - python-version: 3.9
+            slycot: conda
+            array-and-matrix: 1
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install dependencies
+      run: |
+        # Set up conda
+        echo $CONDA/bin >> $GITHUB_PATH
+        conda create -q -n test-environment python=${{matrix.python-version}} 
+        source $CONDA/bin/activate test-environment
+
+        # Set up (virtual) X11
+        sudo apt install -y xvfb
+
+        # Install test tools
+        conda install pip coverage pytest 
+        conda install -c conda-forge pytest-xvfb pytest-cov
+
+        # Install python-control dependencies
+        conda install numpy matplotlib scipy
+        if [[ '${{matrix.slycot}}' == 'conda' ]]; then
+          conda install -c conda-forge slycot
+        fi
+    - name: Test with pytest
+      env:
+        PYTHON_CONTROL_ARRAY_AND_MATRIX: ${{ matrix.array-and-matrix }}
+      run: |
+        source $CONDA/bin/activate test-environment
+        xvfb-run --auto-servernum pytest control/tests
```

Comment:
pytest-xvfb lets you call plain pytest and it starts xvfb-run inside the test suite.

Reply:
I tried that but it was still causing problems with the noX11 tests (they were skipped if I left the current mark in place, and generated an error if I got rid of the mark).  This version seems to work (?).

Reply:
Probably causes the selection of a different mpl backend then

Reply:
Then you don't use and need to install pytest-xvfb

Reply:
Removed pytest-xvfb.